cmake-toolchain/README.md

```diff
@@ -0,0 +1,40 @@
+# Cross Compiliation
```

Comment:
Is there any trouble shooting advice or common errors you can provide? It will help user with debugging.